src/AzureAuth/Commands/CommandInfo.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Commands
+{
+    using System.IO.Abstractions;
+    using System.Reflection;
+    using McMaster.Extensions.CommandLineUtils;
+    using Microsoft.Authentication.AzureAuth.Commands.Info;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Office.Lasso.Telemetry;
+
+    /// <summary>
+    /// The command <see cref="CommandInfo"/> shows debug information and system runtime.
+    /// </summary>
+    [Command(Name = "info", Description = "Shows AzureAuth debug information. Please provide when asking for help.")]
+    [Subcommand(typeof(CommandResetDeviceID))]
+    public class CommandInfo
+    {
+        /// <summary>
+        /// This method executes the info process.
+        /// </summary>
+        /// <param name="logger">The logger.</param>
+        /// <param name="fileSystem">The file system.</param>
+        /// <returns>The error code: 0 is normal execution, and the rest means errors during execution.</returns>
+        public int OnExecute(ILogger<CommandInfo> logger, IFileSystem fileSystem)
+        {
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            string azureauthVersion = assembly.GetName().Version.ToString();
+            string deviceID = TelemetryDeviceID.GetAsync(fileSystem).Result;
+
+            logger.LogInformation(
+                $"AzureAuth Version: {azureauthVersion} \n" +
+                $"Device ID: {deviceID} \n" +
+                $"To reset your device identifier, Run `azureauth info reset-device-id`");
+

```

Comment:
```suggestion
            logger.LogInformation($"AzureAuth Version: {azureauthVersion}");
            logger.LogInformation($"Telemetry Device ID: {deviceID}");
            logger.LogInformation("");
            logger.LogInformation("To reset your device identifier run the following command:");
            logger.LogInformation("  azureauth info reset-device-id");


```

Reply:
This style of using the Logger will ensure the output all makes sense and is readable regardless of what verbosity level is being used. Otherwise you will have messages spanning lines that don't fit nicely with the debug or trace message styles. It also makes it easier to add and remove lines of output in the future.

Anytime you are writing a command that intend someone to copy and paste, put that command on it's own line with at least 2 extra leading spaces and no surrounding qutations.
* the quotes make it hard to select in your terminal
* the extra line and sapces makes it easy to see separate the command from the text, and make it easier to copy since it's not at the border of your CLI so clicking 1-2 characters off doesn't matter.